src/Tizen.Applications.ComponentBased.Port/Tizen.Applications.ComponentBased.Port/ComponentPort.cs

```diff
@@ -139,18 +211,18 @@ public void Send(string endpoint, int timeout, object request)
         }
 
         /// <summary>
-        /// Sends the request data synchronously.
+        /// Sends the request data and receives the reply data.
         /// </summary>
         /// <exception cref="ArgumentException">Thrown when the argument is invalid.</exception>
         /// <exception cref="OutOfMemoryException">Thrown when the memory is insufficient.</exception>
         /// <exception cref="UnauthorizedAccessException">Thrown when because of permission denied.</exception>
         /// <exception cref="global::System.IO.IOException">Thrown when because of I/O error.</exception>
         /// <param name="endpoint">The name of the endpoint</param>
-        /// <param name="timeout">The interval of timeout</param>
+        /// <param name="timeout">The timeout in milliseconds, -1 to use the default timeout</param>
         /// <param name="request">The serializable data to send</param>
         /// <returns>The received serializable data</returns>
         /// /// <since_tizen> 9 </since_tizen>
-        public object SendSync(string endpoint, int timeout, object request)
```

Comment:
Methods should not be removed/replaced like that. Please ensure backward compatibility for applications. The old method should be marked as [Obsolete] and kept for a while (2 API levels). With abstract methods it might be a little trickier to implement, but the methods should be kept as well (as virtual).

Reply:
Oh, this API is not released, It's all API level 9. No issue then.